### DISCUSSION TITLE
posts/2024/01/ambition/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#37] [General]
# posts/2024/01/ambition/

https://guozheng.rbind.io/posts/2024/01/ambition/

**Comment by yihui** [comment]
> 忘记在益辉哪一篇博文中看到，Wordpress（还是其他平台）在加载数千乃至数万份文件时会很慢，甚至会需要几十秒

我是在我的“[关于](https://yihui.org/cn/about/#%E6%9C%AC%E7%AB%99%E6%9E%B6%E6%9E%84)”页面上说的，说的对象是 Jekyll，它预览我自己的网站太慢了。现在已经过去好几年，不知道它是否在速度上有改善。本地预览的速度和部署到服务器上之后的加载速度是两回事，前者取决于网站工具的编译速度，后者取决于服务器的响应速度和网速，与网站编译工具几乎无关。

> 我至少会写上十年的博客

[加油](https://github.com/residualsun1/Residualsun/discussions/9#discussioncomment-6174277)！
